contents/tutorials/nextjs-ab-tests.md

```diff
@@ -1,269 +1,283 @@
 ---
 title: How to set up Next.js A/B tests
-date: 2023-03-06
+date: 2024-02-16
 author: ["ian-vanagas"]
-showTitle: true
-sidebar: Docs
-featuredVideo: https://www.youtube-nocookie.com/embed/QfTMREUEc18
 tags: ['experimentation', 'feature flags', 'actions']
 ---
 
-Many of the features of Next.js optimize for creating the best possible user experience. It automates and abstracts functionality like static page generation, image loading optimization, and optional server-side rendering, allowing you to focus on the content of your app.
+Next.js optimizes for creating the best possible user experience. It automates functionality like static page generation, image optimization, and server-side rendering, enabling you to focus on the content of your app.
 
-A/B tests are a way to make sure this content performs as well as possible. A/B tests involve comparing two or more variations against a goal. The winner is the one which achieves the goal best and is then used to improve everyone's experience.
+[A/B tests](/ab-testing) are a way to make sure this content performs as well as possible. They compare two or more variations on their impact on a goal.
 
-PostHog’s experimentation tool makes this entire process simple. This tutorial shows you how to build a basic Next.js app, add PostHog to it, and finally set up and run an A/B test experiment.
+PostHog's experimentation tool makes this entire process simple. This tutorial shows you how to build a basic Next.js app (with the app router), add PostHog to it, bootstrap feature flag data, and set up the A/B test in the app.
 
-> Already have a Next.js app? [Skip to adding PostHog and setting up the A/B test](#adding-posthog).
```

Comment:
nit: I'd get more to the point with the intro. Something like https://posthog-git-nextjs-ab-test-post-hog.vercel.app/tutorials/django-ab-tests